### PR TITLE
Refresh the MySQL page

### PR DESCRIPTION
**Pull request by YanMeddour**
## Description

Please include a short summary of the changes and what is the purpose of the PR. Any relevant information should be added to help reviewers.

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x
- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] Cloud
- [ ] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ Le connecteur apporte les modèles de service suivants
 |:-------------------|:---------------------------------------|:--------------------------------------------------------------------------------------------------------------------------------|
 | Connection-Time    | App-DB-MySQL-Connection-Time-custom    | Contrôle permettant de vérifier la durée de connexion au serveur. Ce temps est donné en secondes                                |
 | Connections-Number | App-DB-MySQL-Connections-Number-custom | Contrôle permettant de vérifier le nombre de connexions ouvertes                                                                |
-| Database-Size      | App-DB-MySQL-Database-Size-custom      | Contrôle permettant de vérifier la taille des bases de données. Aucune alerte par défaut                                      |
+| Database-Size      | App-DB-MySQL-Database-Size-custom      | Contrôle permettant de vérifier la taille des bases de données. Aucunes alertes par défaut                                      |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:26]:
```suggestion
| Database-Size      | App-DB-MySQL-Database-Size-custom      | Contrôle permettant de vérifier la taille des bases de données. Aucune alerte par défaut                                      |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 2]
```diff
@@ -39,11 +39,11 @@ Le connecteur apporte les modèles de service suivants
...
 | Long-Queries         | App-DB-MySQL-Long-Queries-custom                 | Contrôle permettant de vérifier les requêtes courantes longues                                                                     |
-| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          | Contrôle permettant de vérifier l'état de la réplication entre deux bases de données                                                                                                                                   |
-| Password-Expiration  | App-DB-MySQL-Password-Expiration-custom          | Contrôle permettant de vérifier l'expiration des mots de passe utilisateur                                                       |
+| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          |                                                                                                                                    |
+| Password-Expiration  | App-DB-MySQL-Password-Expiration-custom          | Contrôle permettant de vérifier l'expiration des mots de passe utilisateurs                                                        |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:43]:
```suggestion
| Password-Expiration  | App-DB-MySQL-Password-Expiration-custom          | Contrôle permettant de vérifier l'expiration des mots de passe utilisateur                                                        |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 3]
```diff
@@ -39,11 +39,11 @@ Le connecteur apporte les modèles de service suivants
...
 | Innodb-Bufferpool    | App-DB-MySQL-Innodb-Bufferpool-custom            | Contrôle permettant de vérifier le taux de succès du tampon InnoDB                                                                 |
 | Long-Queries         | App-DB-MySQL-Long-Queries-custom                 | Contrôle permettant de vérifier les requêtes courantes longues                                                                     |
-| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          | Contrôle permettant de vérifier l'état de la réplication entre deux bases de données                                                                                                                                   |
-| Password-Expiration  | App-DB-MySQL-Password-Expiration-custom          | Contrôle permettant de vérifier l'expiration des mots de passe utilisateur                                                       |
+| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          |                                                                                                                                    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:42]:
```suggestion
| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          |   Contrôle permettant de vérifier l'état de la réplication entre deux bases de données                                                                                                                                 |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 4]
```diff
@@ -39,11 +39,11 @@ Le connecteur apporte les modèles de service suivants
...
+| Password-Expiration  | App-DB-MySQL-Password-Expiration-custom          | Contrôle permettant de vérifier l'expiration des mots de passe utilisateurs                                                        |
 | Qcache-Hitrate       | App-DB-MySQL-Qcache-Hitrate-custom               | Contrôle permettant de vérifier le taux d'utilisation du 'query cache'                                                             |
 | Sql-Statement        | App-DB-MySQL-Sql-Statement-Generic-custom        | Contrôle permettant d'exécuter une requête SQL personnalisée renvoyant une donnée numérique                                        |
-| Sql-Statement-String | App-DB-MySQL-Sql-Statement-String-Generic-custom | Contrôle permettant d'exécuter une requête SQL personnalisée renvoyant une chaîne de caractères                                    |
+| Sql-Statement-String | App-DB-MySQL-Sql-Statement-String-Generic-custom | Contrôle permettant d'exécuter une requête SQL personnalisée renvoyant une chaine de caractères                                    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:46]:
```suggestion
| Sql-Statement-String | App-DB-MySQL-Sql-Statement-String-Generic-custom | Contrôle permettant d'exécuter une requête SQL personnalisée renvoyant une chaîne de caractères                                    |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 5]
```diff
@@ -363,8 +363,8 @@ yum install centreon-plugin-Applications-Databases-Mysql
 
 | Macro              | Description                                                                                         | Valeur par défaut                                     | Obligatoire |
 |:-------------------|:----------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|
-| FILTERDATABASE     | Filter by database name (can be a regexp)                                                         | ^(?!(information\_schema\|performance\_schema\|test)) |             |
-| FILTERPERFDATA     | Filter by metric name                                                                    | database                                              |             |
+| FILTERDATABASE     | Filter database to checks (Can be a regexp)                                                         | ^(?!(information\_schema\|performance\_schema\|test)) |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:366]:
```suggestion
| FILTERDATABASE     | Filter by database name (can be a regexp)                                                        | ^(?!(information\_schema\|performance\_schema\|test)) |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 6]
```diff
@@ -363,8 +363,8 @@ yum install centreon-plugin-Applications-Databases-Mysql
...
+| FILTERPERFDATA     |                                                                                                     | database                                              |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:367]:
```suggestion
| FILTERPERFDATA     |      Filter by metric name                                                                                               | database                                              |             |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 7]
```diff
@@ -421,10 +421,10 @@ yum install centreon-plugin-Applications-Databases-Mysql
...
-| CRITICALTHREADIOSTATUS    | Thresholds                                                                                                                                                                                           |                                                   |             |
-| WARNINGTHREADSQLSTATUS    | Thresholds                                                                                                                                                                                           |                                                   |             |
-| CRITICALTHREADSQLSTATUS   | Thresholds                                                                                                                                                                                           |                                                   |             |
+| WARNINGTHREADIOSTATUS     |                                                                                                                                                                                                      |                                                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:424]: description

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 8]
```diff
@@ -671,7 +671,7 @@ Les options disponibles pour chaque modèle de services sont listées ci-dessous
 
 | Option                   | Description                                                                                                            |
 |:-------------------------|:-----------------------------------------------------------------------------------------------------------------------|
-| --filter-database        | Filter by database name (can be a regexp).                                                                           |
+| --filter-database        | Filter database to checks (Can be a regexp).                                                                           |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-mysql.md:674]:
```suggestion
| --filter-database        | Filter by database name (can be a regexp).                                                                           |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 9]
```diff
@@ -38,7 +38,7 @@ The connector brings the following service templates (sorted by the host templat
 | Backup               | App-DB-MySQL-Backup-custom                       | Check backups                                                                                      |
 | Innodb-Bufferpool    | App-DB-MySQL-Innodb-Bufferpool-custom            | Check the hit rate of the InnoDB buffer                                                            |
 | Long-Queries         | App-DB-MySQL-Long-Queries-custom                 | Check current number of long queries                                                               |
-| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          | Check the state of the replication between two databases                                              |
+| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          |                                                                                                    |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:41]:
```suggestion
| MariaDB-Replication  | App-DB-MySQL-MariaDB-Replication-custom          |      Check the state of the replication between two databases                                                                                              |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 10]
```diff
@@ -222,7 +222,7 @@ Listing of user privileges needed by service:
 
 1. If the platform uses an *online* license, you can skip the package installation
 instruction below as it is not required to have the connector displayed within the
-**Configuration > Monitoring Connector Manager** menu.
+**Configuration > Monitoring Connectors Manager** menu.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:225]:
```suggestion
**Configuration > Monitoring Connector Manager** menu.
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 11]
```diff
@@ -258,7 +258,7 @@ yum install centreon-pack-applications-databases-mysql
 </Tabs>
 
 2. Whatever the license type (*online* or *offline*), install the **MySQL/MariaDB** connector through
-the **Configuration > Monitoring Connector Manager** menu.
+the **Configuration > Monitoring Connectors Manager** menu.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:261]:
```suggestion
the **Configuration > Monitoring Connector Manager** menu.
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 12]
```diff
@@ -357,30 +357,30 @@ yum install centreon-plugin-Applications-Databases-Mysql
...
+| EXTRAOPTIONS      | Any extra option you may want to add to the command (e.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
 
 </TabItem>
 <TabItem value="Database-Size" label="Database-Size">
 
 | Macro              | Description                                                                                         | Default value                                         | Mandatory   |
 |:-------------------|:----------------------------------------------------------------------------------------------------|:------------------------------------------------------|:-----------:|
-| FILTERDATABASE     | Filter by database name (can be a regexp)                                                         | ^(?!(information\_schema\|performance\_schema\|test)) |             |
+| FILTERDATABASE     | Filter database to checks (Can be a regexp)                                                         | ^(?!(information\_schema\|performance\_schema\|test)) |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:367]:
```suggestion
| FILTERDATABASE     | Filter by database name (can be a regexp)                                                        | ^(?!(information\_schema\|performance\_schema\|test)) |             |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 13]
```diff
@@ -357,30 +357,30 @@ yum install centreon-plugin-Applications-Databases-Mysql
...
-| CRITICALTOTALFREE  | Thresholds                                                                                          |                                                       |             |
-| WARNINGTOTALUSAGE  | Thresholds                                                                                          |                                                       |             |
-| CRITICALTOTALUSAGE | Thresholds                                                                                          |                                                       |             |
-| EXTRAOPTIONS       | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) | --verbose                                             |             |
+| WARNINGDBFREE      | Thresholds (                                                                                        |                                                       |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:369]: remove all parentheses

File: pp/integrations/plugin-packs/procedures/applications-databases-mysql.md [hunk 14]
```diff
@@ -670,7 +670,7 @@ All available options for each service template are listed below:
 
 | Option                   | Description                                                                                                            |
 |:-------------------------|:-----------------------------------------------------------------------------------------------------------------------|
-| --filter-database        | Filter by database name (can be a regexp).                                                                           |
+| --filter-database        | Filter database to checks (Can be a regexp).                                                                           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-mysql.md:673]:
```suggestion
| --filter-database        | Filter by database name (can be a regexp).                                                                           |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

**Comment by omercier** [comment]
We will update the Mysql doc later, this PR would make more harm than good.

**Pull request closed without merging**